experimenter/experimenter/nimbus_ui_new/views.py

```diff
@@ -252,11 +270,71 @@ def post(self, *args, **kwargs):
         return response
 
 
+class NimbusExperimentsCloneView(NimbusExperimentViewMixin, RequestFormMixin, CreateView):
+    def get_form_kwargs(self):
+        kwargs = super().get_form_kwargs()
+        kwargs["data"] = kwargs["data"].copy()
+        kwargs["data"]["owner"] = self.request.user
+        kwargs["parent_slug"] = self.kwargs.get("slug")
+        kwargs["branch_slug"] = self.kwargs.get("branch")
+        return kwargs
+
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+        context["experiment"] = self.get_object()
+        return context
```

Comment:
Why is this necessary?  Shouldn't this happen automatically?

Reply:
:+1: because of `NimbusExperimentViewMixin` settings `model` and `context_object_name`, this should be unnecessary.

Reply:
Yeah I tried it too, we coudln't figure out why it's necessary but no need to block here if it works, filed as a followup

https://github.com/mozilla/experimenter/issues/12432